ldms/src/test/Makefile.am

```diff
@@ -41,10 +41,14 @@ test_metric_LDFLAGS = $(AM_LDFLAGS) -pthread -lm
 
 # override pkglib sanity checks
 mypkglibdir = $(pkglibdir)
-mypkglib_SCRIPTS = ldms-run-static-tests.sh
+mypkglib_SCRIPTS = ldms-run-static-tests.test
 mypkglib_DATA = ldms-static-test-list.sh ldms-static-test-bypass
 
+# invoke tap-driver
+TEST_LOG_DRIVER = env AM_TAP_AWK='$(AWK)' $(SHELL) \
+                  $(top_srcdir)/config/tap-driver.sh
+
 installcheck-local: 
-	ldms-run-static-tests.sh
+	ldms-run-static-tests.test.in
```

Comment:
@Snell1224 installcheck-local here is still failing when I run make installcheck. 
It may be you need to include ./ in front of the command, since . is not in the PATH by default.

Do we need both check and installcheck here (do they do something different?). If not, we can just remove installcheck-local.

Reply:
@baallan Yes, you're right, make installcheck did not work for me either. I realized that I had forgot to "enable-ldms-test" when running make installcheck earlier. 

I included the ./, enabled the ldms-test option, and installcheck worked. I've already made the changes to this branch so you can look it over again. 

About the check and installcheck, yes I believe they both will be needed. Some of Jenkins test scripts test with make check and others with make installcheck (install-test.sh and unit-test.sh). Those that use make installcheck will fail whenever the 'ldms-test' option is enabled. Yes, they are slightly different in that they test different parts of the program. Make check tests the files in the source tree while the make installcheck tests the their installed copies. So, I think it might be easier/better to keep both.